test/integration/test-amp-bind.js

```diff
@@ -618,4 +618,31 @@ describe.configure().ifNewChrome().run('amp-bind', function() {
           });
         });
   });
+
+  // The only difference in amp4email is that URL attributes cannot be bound.
+  describe('amp4email', () => {
+    beforeEach(() => {
+      return setupWithFixture('test/fixtures/bind-amp4email.html');
```

Comment:
nitpick, can we have another testcase with the same contents of this html doc where the AMP4Email format type is declared as 'amp4email' rather than '⚡4email'